joplin/templates/wagtailadmin/pages/search_header.html

```diff
@@ -10,13 +10,23 @@ <h1 class="coa-search-results-h1">
 
     <div>
         <!-- From wagtail/admin/templates/wagtailadmin/shared/menu_search.html -->
-        <form class="nav-search" action="{% url 'wagtailadmin_pages:search' %}" method="get">
+        {% if search_url %}
+            <form class="col search-form nav-search" action="{% url search_url %}{% if query_parameters %}?{{ query_parameters }}{% endif %}" method="get" novalidate>
```

Comment:
this form does ajax magic. i still don't fully understand why. but i know its one of the main reasons why we wanted to move to search page